Infrastructure/Bootstrapper.cs

```diff
@@ -0,0 +1,137 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.AzureAD.UI;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Authorization;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.IdentityModel.Protocols.OpenIdConnect;
+using WebApp_OpenIDConnect_DotNet.Interfaces;
+using WebApp_OpenIDConnect_DotNet.Services;
+
+namespace WebApp_OpenIDConnect_DotNet.Infrastructure
+{
+    public static class Bootstrapper
+    {
+        public static void InitializeDefault(this IServiceCollection services, IConfiguration configuration)
```

Comment:
Good point, you are suggesting that the Startup.cs should be even simpler.
I got this feedback already as well.

Any objections if I moved this file and the files under Extensions to a new library?

Reply:
No objections, feel free to moved it as you wish 👍